napari/_tests/test_pytest_plugin.py

```diff
@@ -30,3 +30,32 @@ def test_make_viewer(make_napari_viewer):
 
     # check that all 1 test passed
     result.assert_outcomes(passed=1)
+
+
+def test_napari_plugin_tester(testdir):
+    """Make sure that our napari_plugin_tester fixture works."""
+
+    # create a temporary pytest test file
+    testdir.makepyfile(
+        """
+        from napari_plugin_engine import napari_hook_implementation
+        class Plugin:
+
+            @napari_hook_implementation
+            def napari_get_reader(path):
+                pass
+
+        def test_pm(napari_plugin_tester):
+            napari_plugin_tester.register(Plugin)
+            napari_plugin_tester.assert_plugin_name_registered("Plugin")
+            napari_plugin_tester.assert_module_registered(Plugin)
+            napari_plugin_tester.assert_implementations_registered(
+                Plugin, "napari_get_reader"
+            )
+    """
+    )
+    # run all tests with pytest
+    result = testdir.runpytest()
```

Comment:
Jeez sometimes I really hate pytest magic... Is testdir a magical fixture that you get automatically with pytest?

Reply:
I know... it comes from pytest itself, and is provided to test plugins: https://docs.pytest.org/en/stable/writing_plugins.html#testing-plugins

Reply:
I just saw that pytester is only available in 6.2 (?) — do we want to add that minimum version to our setup.cfg?

Reply:
sure I can add that

Reply:
as discussed on zulip, "pytester" the _fixture_ is new as of 6.2 ... but pytester the _plugin_ (with the `testdir` fixture) has been around for a long time.